bayesflow/helper_functions.py

```diff
@@ -109,12 +110,12 @@ def extract_current_lr(optimizer):
         Current learning rate, or `None` if it can't be determined
     """
 
-    if isinstance(optimizer.lr, LearningRateSchedule):
+    if isinstance(optimizer.learning_rate, LearningRateSchedule):
         # LearningRateSchedule instances need number of iterations
-        current_lr = optimizer.lr(optimizer.iterations).numpy()
-    elif hasattr(optimizer.lr, "numpy"):
+        current_lr = optimizer.learning_rate(optimizer.iterations).numpy()
```

Comment:
not sure if this works under jax/numpy backend - can you test this? As an alternative, we could use `keras.ops.convert_to_numpy()`. We can then drop the `hasattr` check

Reply:
Changed `learning_rate().numpy()` to `keras.ops.convert_to_numpy()` in helper_functions.py (see [47fbbbd](https://github.com/stefanradev93/BayesFlow/pull/165/commits/47fbbbd4acb22c391b483c845f73e6862094c07a))